entrypoint.sh

```diff
@@ -72,29 +85,47 @@ echo "$SUPER_USER_PASSWORD" > $FACTORY_DATA/root.pwd
 
 
 case "$DBMS_TYPE" in
-    "mariadb") 
+    "mariadb")
         DB_PORT="3306"
+        check_db_access
+
+        db_exist=$(mysql -h $DB_HOST -u $DB_USER -p$DB_PASS -e "show databases like 'coucou'")
+        if [ "$db_exist" == "" ]; then
+            echo "Database doesn't exist. Trying to create it..."
+            mysql -h $DB_HOST -u $DB_USER -p$DB_PASS -e "create database $DB_NAME"
+        fi
+
+        tables_list=$(mysql -h $DB_HOST -u $DB_USER -p$DB_PASS -e "show tables" $DB_NAME)
+        if [ "$tables_list" == "" ]; then
+            echo "Database is empty. Going to initialize the database schema..."
+            mysql -h $DB_HOST -u $DB_USER -p$DB_PASS $DB_NAME <<< $(cat /data/digital-factory-data/db/sql/schema/mysql/01*)
+        fi
+
         testdb_result="$(mysql -u $DB_USER -p$DB_PASS -h $DB_HOST -D $DB_NAME -e "select count(REVISION_ID) from JR_J_LOCAL_REVISIONS;" -s)"
         ;;
     "postgresql")
         DB_PORT="5432"
+        check_db_access
+
+        db_exists=$(PGPASSWORD=$DB_PASS psql -t -h $DB_HOST -U $DB_USER -d postgres -c "SELECT datname FROM pg_database where datname='$DB_NAME';")
+        if [ "$db_exists" == "" ]; then
+            echo "Database doesn't exist. Trying to create it..."
+            PGPASSWORD=$DB_PASS psql -h $DB_HOST -U $DB_USER -d postgres -c "create database $DB_NAME"
```

Comment:
```suggestion
            PGPASSWORD=$DB_PASS psql -h $DB_HOST -U $DB_USER  -c "create database $DB_NAME"
```


have to be sure that the database is created in order to continue or break too

Reply:
Could find a solution to remove  this " -d postgres"